### ISSUE TITLE
chruby detection broken since 0.5.13

### ISSUE THREAD

**Issue by tvdeyen** [Shopify/ruby-lsp#1603]
This broke chruby detection for me. I **HAVE** a `.ruby-version` in my repo (a gem, not a Rails app) and I have a global `~/.ruby-version` file in my home folder. It works everywhere, but not with this version of the ruby-lsp extension in latest VSCode. It was working perfectly until this change.

_Originally posted by @tvdeyen in https://github.com/Shopify/ruby-lsp/issues/1438#issuecomment-1994621516_

**Comment by tvdeyen** [comment]
This is how you pin to a previous version https://stackoverflow.com/questions/42626065/vs-code-how-to-rollback-extension-install-specific-extension-version

**Comment by sweetyclem** [comment]
I'm also getting a chruby error, I think this is the same issue
```
Failed to activate chruby environment: Cannot find installation directory for Ruby version 3.2.2
```

**Comment by reid-rigo** [comment]
I'm on macOS 14.4

```sh
> which ruby
/Users/me/.rubies/ruby-3.2.2/bin/ruby
```

**Comment by reid-rigo** [comment]
**Ruby version:**
```sh
> cat .ruby-version
3.2.2
```

**Extension output:**
```log
2024-03-13 12:50:15.250 [info] (wb-app) Checking if asdf is available on the path with command: /bin/zsh -i -c 'asdf --version'
2024-03-13 12:50:15.856 [info] (wb-app) Checking if chruby is available on the path with command: /bin/zsh -i -c 'chruby --version'
2024-03-13 12:50:16.402 [info] (wb-app) Discovered version manager chruby
2024-03-13 12:50:16.413 [info] (wb-app) Discovered Ruby version 3.2.2 from file:///Users/me/app/.ruby-version
```
Followed by the error `Failed to activate chruby environment: Cannot find installation directory for Ruby version 3.2.2`

**Permissions:**
`~/.rubies` is owned by my user

**Comment by tvdeyen** [comment]
> Thank you for the report. Are you installing your rubies in a custom directory by any chance? For example, modifying the `RUBIES` environment variable so that `chruby` can find them in a non standard place? Please let us know where your rubies are installed.

No. The rubies are in `~/.rubies` what is the default I think.

> Also, what operating system are you using?

I am on macOS 14.3.1 (23D60)

**Comment by vinistock** [comment]
Just published [v0.5.14](https://github.com/Shopify/ruby-lsp/releases/tag/vscode-ruby-lsp-v0.5.14). Please let us know if the issue is fixed.

**Comment by tvdeyen** [comment]
Works like charm! Thanks
